.github/workflows/perf-runtime.yml

```diff
@@ -0,0 +1,147 @@
+name: Perf - Runtime
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: '*'
+
+jobs:
+  benchmark:
+    strategy:
+      fail-fast: false
+      # `matrix` creates a job matrix (Cartesian product of possible values for each var)
+      # https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix
+      matrix:
+        component:
+          - locale
+          - num-util
+          - uniset
+      # Ensure that commits when pushing GH pages to external has linear commit
+      # history w/o merges (to keep things simpler). Do this via max-parallel=1.
+      # Also, necessary b/c can't figure out how to use component-specific branches
+      # and coordinate between the benchmark action step and the GH pages action step.
+      max-parallel: 1
```

Comment:
do you think it's better over `cargo bench --all-features` from main `icu4x` directory?